frontend/src/tests/applicant-tests.js

```diff
@@ -126,7 +126,67 @@ export function applicantTests(api) {
         expect(resp.payload).toContainObject(newApplicant);
     });
 
-    it.todo("delete an applicant with no associated assignments");
+    it("delete an applicant with no associated assignments", async () => {
+        // upsert and delete an applicant without a session
+        const newApplicant1 = {
+            first_name: "June",
+            last_name: "Kim",
+            utorid: "kim",
+        };
+        const resp = await apiPOST(`/admin/applicants`, newApplicant1);
+        expect(resp).toHaveStatus("success");
+        Object.assign(newApplicant1, resp.payload);
+
+        // try deleting the applicant
+        const resp2 = await apiPOST(`/admin/applicants/delete`, {
```

Comment:
Unless you actually need it, do one `let resp` at the beginning and then reuse `resp` throughout.